src/main/scala/uk/gov/hmrc/ct/ct600/v3/calculations/B325Calculator.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.ct.ct600.v3.calculations
+
+import uk.gov.hmrc.ct.box.CtTypeConverters
+import uk.gov.hmrc.ct.computations.CP291
+import uk.gov.hmrc.ct.ct600.v3._
+
+trait B325Calculator extends CtTypeConverters {
```

Comment:
It seems that most of single calculations are closer to boxes instead of embedding them in a trait.
I might missed something, tho

Reply:
Just trying to follow the code convention since the calculations for the CP has been done through the trait but if you want i can put these calculation as part of B325 rather than embedding them in a trait.

Reply:
Which convention? Each calculator had a lot of formulas. There is no single method calculators in the code.